tsconfig.json

```diff
@@ -14,8 +15,7 @@
     "jsx": "react",
     "lib": [
       "dom",
-      "es5",
-      "es2015"
+      "ESNext"
```

Comment:
As I pointed out in [FOSS #6](https://github.com/concord-consortium/foss-simulations/pull/6):

> My original comment on PR #5 was that `"lib": ["dom", "es5", "es2015", "es2016"]` could be replaced with `"lib": ["dom", "es2016"]` since the `esxxxx` values are cumulative. Switching to `ESNext` here suggests that we're expecting to use the absolute most recent JS changes, some of which may not be implemented in our users' browsers yet and so we would have to make sure that we're applying appropriate polyfills, which I'm not sure is true in all of our projects. My thinking on the specific value here is that we should either choose a somewhat conservative value that we expect our users' browsers to meet (2017? 2018?) or expect to polyfill for anything newer. In CLUE we're using [ts-polyfill](https://www.npmjs.com/package/ts-polyfill) which was originally billed as a polyfill library that would keep current with each new TypeScript/JavaScript release, but now hasn't been updated in a couple years so maybe we need to look for a new polyfill solution. TL/DR: I would think that standardizing on a more conservative value for `lib` would make more sense, but I'm not entirely sure what that should be. @scytacki might have an opinion on what expectations we have for our users' browsers.